### DISCUSSION TITLE
Z position on Bed Level panel

### DISCUSSION THREAD

**Discussion by jessbo** [KlipperScreen/KlipperScreen#1037] [Show and tell]
Hello, I made a small change to the bed leveling panel in my fork (jessbo/klipperscreen@0a77bb0) and wanted to share.

![Needs adjustment](https://raw.githubusercontent.com/jessbo/klipperscreen/verbose-screws_tilt_adjust/adjust.png)
![Level](https://raw.githubusercontent.com/jessbo/klipperscreen/verbose-screws_tilt_adjust/level.png)

**Comment by alfrix** [comment]
i appreciate the effort, but not sure about adding special characters, they have been known to cause issues in the past: #770 and #254

**Comment by alfrix** [comment]
Current version of ks changes the icon instead of using special characters, for z there is an FR opened
